meshio/abaqus/_abaqus.py

```diff
@@ -92,8 +92,39 @@
 meshio_to_abaqus_type = {v: k for k, v in abaqus_to_meshio_type.items()}
 
 
+def bulk_w_includes(inp_path, bulk_str):
+    import os
+    import re
+
+    re_in = re.compile(
+        r"\*Include,\s*input=(.*?)$", re.IGNORECASE | re.MULTILINE | re.DOTALL
+    )
+    bulk_repl = dict()
+
+    for m in re_in.finditer(bulk_str):
+        search_key = m.group(0)
+        with open(os.path.join(os.path.dirname(inp_path), m.group(1)), "r") as d:
```

Comment:
Better use pathlib.Path.

Reply:
Sure! Pathlib is new to me. Would something like this be okay? 

```python
from pathlib import Path
str(Path(inp_path, m.group(1)).absolute())
```

Reply:
No need for `str`. Also, use `/` for joining the paths.